src/Sleep.js

```diff
@@ -0,0 +1,103 @@
+class Sleep {
+  constructor(sleepData) {
+    this.sleepData = sleepData
+  }
+
+  calculateDataAverage(userID, dataMetric) {
+    const currentUserSleepData = [];
+    const total = this.sleepData.reduce((sum, sleepInfo) => {
+      if (sleepInfo.userID === userID) {
+        currentUserSleepData.push(sleepInfo)
+        sum += sleepInfo[dataMetric];
+      }
+      return sum;
+    }, 0);
+    return Math.round(total / currentUserSleepData.length);
+  }
```

Comment:
Can we make this more dynamic by taking in the data Array we'll want to refer to?
It looks like we had to do something similar on lines 76 - 80